setup.py

```diff
@@ -40,8 +39,9 @@
         "torchvision>=0.5.0",
```

Comment:
outdated?

Reply:
i think that torch would supersede this anyway, but it can be updated as well

Reply:
i updated it to match the requirements of torch 1.7

Reply:
I mean all listed requirements. Many of them are not working with python3.7+. Only the last line came up here.

Reply:
which ones dont work with 3.7?

Reply:
numpy scipy mpi4py pillow

Reply:
your package manager should take care of which is the best implementation which fits these requirements for your python version